android/app/src/main/res/layout/item_speaker.xml

```diff
@@ -2,6 +2,8 @@
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
+    android:elevation="6dp"
```

Comment:
This won't work for lower version APIs instead, put this in a CardView and use cardElevation in that

Reply:
@heysadboy Okay!